src/test/PoolSwapTest.sol

```diff
@@ -59,35 +62,47 @@ contract PoolSwapTest is Test, PoolTestBase {
         (,, uint256 reserveAfter1, int256 deltaAfter1) = _fetchBalances(data.key.currency1, data.sender);
 
         // Make sure youve added liquidity to the test pool!
-        if (BalanceDelta.unwrap(delta) == 0) revert NoSwapOccurred();
-
-        assertEq(reserveBefore0, reserveAfter0);
-        assertEq(reserveBefore1, reserveAfter1);
-
-        if (data.params.zeroForOne) {
-            if (data.params.amountSpecified > 0) {
-                // exact input, 0 for 1
-                assertEq(deltaAfter0, data.params.amountSpecified);
-                assert(deltaAfter1 < 0);
+        if (BalanceDelta.unwrap(delta) == 0) revert NoSwapOccurred(); // Note: Delta actually can end up being 0 even if there was liquidity originally intialized. This happens when liquidity gets pushed entirely into 1 token.
```

Comment:
is there an action item on this note?

Reply:
Hm no.. I can remove the note and honestly just remove this check though. We already remove it in Alice's NoOp Pr